lib/routes/people-routes.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const app = express();
+
+
+const valid = () => {
+  return (req, res, next) => {
+    req.valid = Math.random() >= 0.5;
+    next();
+  };
+};
+// have a function that sets req.message called "setMessage"
+const setMessage = (str) => {
+  return (req, res, next) => {
+    req.setMessage = 'str';
```

Comment:
You're setting the message equal to the literal value `'str'` instead of the variable `str`